src/agent/driver-test-runner.js

```diff
@@ -153,6 +121,11 @@ export class DriverTestRunner {
           },
         });
       }
+    } else if (atName == 'Voiceover') {
+      if (settings !== 'defaultMode') {
+        throw new Error(`Unrecognized setting for VoiceOver: ${settings}`);
+      }
```

Comment:
Would it be possible to enable these [settings](https://github.com/w3c/aria-at/blob/625a22e13bd117c4bb8f7ecdb5e97454f9e935ac/tests/support.json#L65:L108) via AppleScript?

Reply:
Good question! I just checked the entry for VoiceOver in the macOS Script Editor "Library" (I'd link to it here, but I can't find an online reference), but there doesn't appear to be any API related to modifying settings.

This will be a problem if/when the CG wishes to write tests for behavior when one of these settings does not have its default value. As I understand things, though, the CG is currently limiting their scope to the screen reader's default settings. If that's the case, then lack of scripting support should be fine for now.

Can you confirm, @howard-e?

Reply:
> As I understand things, though, the CG is currently limiting their scope to the screen reader's default settings.

Well not as of recently. There are now 28 test instances where default mode isn't being used with VO. This isn't in all test suites as yet though. I ran the following command in [w3c/aria-at](https://github.com/w3c/aria-at):

```bash
find tests -type f -name "voiceover_macos-commands.csv" -exec awk -F',' '$3 != null && $3 != "settings" {print FILENAME ":" $0}' {} \;
```

<details>

<summary>28 results</summary>

```bash
tests/link-span-text/data/voiceover_macos-commands.csv:navForwardsToLink,l ,singleQuickKeyNavOn,,3.2
tests/link-span-text/data/voiceover_macos-commands.csv:navBackToLink,shift+l,singleQuickKeyNavOn,,6.2
tests/command-button/data/voiceover_macos-commands.csv:navForwardsToButton,b,singleQuickKeyNavOn,,5.3
tests/command-button/data/voiceover_macos-commands.csv:navForwardsToButton,j ,singleQuickKeyNavOn,,5.2
tests/command-button/data/voiceover_macos-commands.csv:navBackToButton,shift+b,singleQuickKeyNavOn,,6.3
tests/command-button/data/voiceover_macos-commands.csv:navBackToButton,shift+j,singleQuickKeyNavOn,,6.2
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navForwardsInToRadioGroupWhereNoRadioButtonsAreChecked,j,singleQuickKeyNavOn,,2.2
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navBackInToRadioGroupWhereNoRadioButtonsAreChecked,shift+j,singleQuickKeyNavOn,,4.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navForwardsInToRadioGroupWhereFirstRadioButtonIsChecked,j,singleQuickKeyNavOn,,6.2
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:NavBackIntoRadioGroupWhereLastRadioChecked,shift+j,singleQuickKeyNavOn,,8.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navOutStartRadioGroup,shift+l,singleQuickKeyNavOn,3:groupBoundary ,10.2
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navOutEndRadioGroup,l,singleQuickKeyNavOn,3:groupBoundary ,12.2
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToNextUncheckedRadio,j,singleQuickKeyNavOn,,20.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToPrevUncheckedRadio,shift+j,singleQuickKeyNavOn,,22.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToNextCheckedRadio,j,singleQuickKeyNavOn,,24.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToPrevCheckedRadio,shift+j,singleQuickKeyNavOn,,26.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToFirstRadio,down,arrowQuickKeyNavOff,,30
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToFirstRadio,right,arrowQuickKeyNavOff,,30.1
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToLastRadio,up,arrowQuickKeyNavOff,,32
tests/radiogroup-aria-activedescendant/data/voiceover_macos-commands.csv:navToLastRadio,left,arrowQuickKeyNavOff,,32.1
tests/toggle-button/data/voiceover_macos-commands.csv:navForwardsToNotPressedToggleButton,b,singleQuickKeyNavOn,,5.3
tests/toggle-button/data/voiceover_macos-commands.csv:navForwardsToNotPressedToggleButton,j ,singleQuickKeyNavOn,,5.2
tests/toggle-button/data/voiceover_macos-commands.csv:navBackToNotPressedToggleButton,shift+b,singleQuickKeyNavOn,,6.3
tests/toggle-button/data/voiceover_macos-commands.csv:navBackToNotPressedToggleButton,shift+j ,singleQuickKeyNavOn,,6.2
tests/toggle-button/data/voiceover_macos-commands.csv:navForwardsToPressedToggleButton,b,singleQuickKeyNavOn,,11.3
tests/toggle-button/data/voiceover_macos-commands.csv:navForwardsToPressedToggleButton,j ,singleQuickKeyNavOn,,11.2
tests/toggle-button/data/voiceover_macos-commands.csv:navBackToPressedToggleButton,shift+b,singleQuickKeyNavOn,,12.3
tests/toggle-button/data/voiceover_macos-commands.csv:navBackToPressedToggleButton,shift+j ,singleQuickKeyNavOn,,12.2
```

</details>

>  If that's the case, then lack of scripting support should be fine for now.

Even though there are now instances, this should still be fine for the moment. It's certainly a current limitation that hasn't yet been explored, but considerations for this should now be reflected in the app as well.